src/test/java/seedu/address/logic/commands/SortCommandTest.java

```diff
@@ -23,11 +23,13 @@
 import seedu.address.model.UserPrefs;
 import seedu.address.model.person.DateDistantToRecentComparator;
 import seedu.address.model.person.DateRecentToDistantComparator;
+import seedu.address.model.person.PriorityHighToLowComparator;
+import seedu.address.model.person.PriorityLowToHighComparator;
 
 /**
  * Contains integration tests (interaction with the Model) for {@code SortByDateCommand}.
```

Comment:
@YINZE66 For this part here I think you meant `... for {@code SortByCommand}`

Reply:
yep